src/dbconfig/dbconfig.ts

```diff
@@ -0,0 +1,12 @@
+import * as pg from 'pg';
+const Pool = pg.Pool;
+
+export default new Pool (    {
```

Comment:
don't expose this directly, use a custom wrapper around it, so that in case you have to change libraries tomorrow you wouldn't have to change the rest of the code only this file and it's interface

Reply:
using prisma now, this is no longer exposed :)